css-brochure-form/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div>
+
+    <h2>LEARN MORE</h2>
+
+    <form action="" method="get">
+      <p>
+        <label for="email">COMPANY EMAIL</label><br>
+        <input class="fill" type="text" name="email" id="email" placeholder="sally@company.co">
+      </p>
+
+      <p>
+        <label for="name">NAME</label><br>
+        <input class="fill" type="text" name="name" id="name" placeholder="Sally Smith">
+      </p>
+
+      <p>
+        <label for="location">WHERE ARE YOU HIRING?</label><br>
+        <input class="fill" type="text" name="location" id="location" placeholder="City, Country">
+      </p>
+
+      <p>
+        <label for="number">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label><br>
+        <select class="fill" name="number" id="number">
+          <option class="fill" value="1-2">1-2 Engineers</option>
+          <option class="fill" value="3-4">3-4 Engineers</option>
+        </select>
+      </p>
+
+      <p class="sub-btn">
+        <input class="submit" type="submit" name="submit" id="submit" value="SUBMIT">
+      </p>
```

Comment:
You should use `div` elements here rather than `p` elements to be a container for your inputs. `p` is for default text, `div` is our go-to element for making a container around a group of elements